### DISCUSSION TITLE
Signal-cli + Signal-Desktop

### DISCUSSION THREAD

**Discussion by paolo48** [AsamK/signal-cli#783] [General]
upfront.long time solydx user but not a tech person.  so i found this link.

and i followed it process to this point...where i am stuck. took me a few tries to get my phone number right. :o

signal-cli-0.6.2/lib/core-1.51.0.0.jar
signal-cli-0.6.2/lib/debug-1.1.1.jar
signal-cli-0.6.2/lib/hexdump-0.2.1.jar
signal-cli-0.6.2/lib/curve25519-java-0.4.1.jar
signal-cli-0.6.2/bin/
signal-cli-0.6.2/bin/signal-cli.bat
signal-cli-0.6.2/bin/signal-cli
paolo48@SolydXK:~$ cd signal-cli-0.6.2
paolo48@SolydXK:~/signal-cli-0.6.2$ sudo bin/signal-cli -u  deleted number
usage: signal-cli [-h] [-v] [--config CONFIG] [-u USERNAME | --dbus |
--dbus-system]
{trust,receive,listIdentities,link,unregister,updateGroup,quitGroup,listDevices,removeDevice,updateAccount,daemon,setPin,addDevice,listGroups,verify,send,removePin,register}
...
signal-cli: error: invalid choice: '-' (choose from 'trust', 'receive',
'listIdentities', 'link', 'unregister', 'updateGroup', 'quitGroup',
'listDevices', 'removeDevice', 'updateAccount', 'daemon', 'setPin',
'addDevice', 'listGroups', 'verify', 'send', 'removePin', 'register')

Did you mean:
link
send
paolo48@SolydXK:~/signal-cli-0.6.2$

**Comment by technillogue** [comment]
You're probably referring to https://ctrl.alt.coop/en/post/signal-without-a-smartphone/ ; your link is broken. You haven't posted enough of your logs for me to understand, but it looks like you're not putting in the command at the end. You don't actually have to run signal-cli in demon mode for this to work; all you need to do is register and then use the `addDevice` command with the URI from the Signal-Desktop code. You can also just use a screenshot tool and any scanner to get that code, you don't need developer tools and zbar-img specifically

**Comment by paolo48** [comment]
txs for that. yes i was a bit sloppy with my post. 

here is the link https://levelup.gitconnected.com/how-to-install-signal-without-a-phone-33a402cd318e

i've managed to, recently, connect signal on my desktop using android studio. this was so much easier for me than trying to fiddle around in terminal.  i wasn't comfortable there. 

in any case i was planning to come back to say my issue has been resolved when i received an email that someone had posted a comment. so thank you again for that. 

now i'm not quite sure if there is a button or something else where i can mark this issue resolved.
